vg/vgsvg/vgsvg.go

```diff
@@ -39,7 +35,7 @@ const (
 type Canvas struct {
 	svg  *svgo.SVG
 	w, h vg.Length
-	dpi  float64
+	// dpi  float64
```

Comment:
remove?

Reply:
Thanks

Reply:
BTW Do you know what `ht` is for. It is not used AFAICS.

Reply:
no idea :)
according to blame, @eaburns is the one who introduced it:
https://github.com/gonum/plot/blame/07f44254b21cff40a59809f4c926aa023286107d/vg/vgsvg/vgsvg.go